releases.yml

```diff
@@ -1,4 +1,39 @@
 releases:
+  ec.2:
+    assembly:
+      basis:
+        brew_event: 62525080
+        reference_releases:
+          x86_64: 4.19.0-0.nightly-2025-02-11-035912
```

Comment:
This i think needs to be commented out, otherwise the pipeline will complain about other arches